src/components/line/filterToolbox/timeAggregation.js

```diff
@@ -0,0 +1,58 @@
+import React, { memo, useMemo } from "react"
+import Menu from "@netdata/netdata-ui/lib/components/drops/menu"
+import { useAttributeValue, useChart } from "@/components/provider"
+import Label from "./label"
+
+const useItems = chart =>
```

Comment:
Maybe a more descriptive name here will be better. Also i question here the `useMemo`.

Reply:
On it! 😄